themes/default/theme.lua

```diff
@@ -1,6 +1,7 @@
 ---------------------------
 -- Default awesome theme --
 ---------------------------
+-- luacheck: no check comment
```

Comment:
I am not sure what to do about that one. I think there is a way to omit some warnings in the `.luacheckrc`, maybe this is a better option for this one? @blueyed: opinion?